website/docs/d/pi_host.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+subcategory: "Power Systems"
+layout: "ibm"
+page_title: "IBM: ibm_pi_host"
+description: |-
+  Get information about a host in Power Virtual Server.
+---
+
+# ibm_pi_host
+
+Provides a read-only data source to retrieve information about a host. For more information, about IBM power virtual server cloud, see [getting started with IBM Power Systems Virtual Servers](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-getting-started).
+
+## Example Usage
+
+```terraform
+data "ibm_pi_host" "host" {
+    pi_cloud_instance_id  = "<value of the pi_cloud_instance_id>"
+    pi_host_id = "<value of the pi_host_id>"
+}
+```
+
+## Notes
+
+- Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
+- If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
+  - `region` - `lon`
+  - `zone` - `lon04`
+  
+Example usage:
+
+  ```terraform
+    provider "ibm" {
+      region    =   "lon"
+      zone      =   "lon04"
+    }
+  ```
+
+## Argument Reference
+
+You can specify the following arguments for this data source.
+
+- `pi_cloud_instance_id` - (Required, String) The GUID of the service instance associated with an account.
+
+- `pi_host_id` - (Required, Forces new resource, String) Host ID.
```

Comment:
So ```ForceNew:    true,``` isn't set on this property, should it be set, or should this description be updated to remove "force new resource"? It's kind of implied that removing the id of your resource will cause the resource to be recreated, but we don't mark cloud_instance_id as force new.

Reply:
It is not needed here.